src/shared-testing/src/FakeS3Gateway.ts

```diff
@@ -31,6 +31,15 @@ export default class FakeS3Gateway implements S3GatewayInterface {
     return Promise.resolve(this.items[key])
   }
 
+  doesItemExist(key: string): PromiseResult<boolean> {
+    if (this.error) {
+      return Promise.resolve(this.error)
+    }
+
+    // eslint-disable-next-line no-prototype-builtins
+    return Promise.resolve(this.items.hasOwnProperty(key))
```

Comment:
You could probably get rid of the comment on line 39 and use `Boolean(this.items[key])` or the "in" operator?

Reply:
Thanks Jazz! Updated.